ExilenceNextApp/src/services/poe-ninja.service.ts

```diff
@@ -50,6 +50,12 @@ function getItemCategories() {
     'Beast',
     'Vial',
     'Invitation',
+    'Artifact',
+    'UniqueJewel',
+    //'ClusterJewel',
```

Comment:
Reason for commenting this out?

Reply:
I added it in commit msg. So basically those are usually medium confidence and sometimes misleading, so I decided to comment it out. I can add it back if you think its make sense.

Reply:
Alright, I think we should keep them for now at least, then decide what we do with them for the next league

Reply:
updated